corehq/apps/app_manager/views/modules.py

```diff
@@ -196,16 +197,20 @@ def _report_to_config(report):
     all_reports = ReportConfiguration.by_domain(app.domain) + \
                   StaticReportConfiguration.by_domain(app.domain)
     warnings = []
-    validity = module.check_report_validity()
-
-    # We're now proactively deleting these references, so after that's been
-    # out for a while, this can be removed (say June 2016 or later)
-    if not validity.is_valid:
-        module.report_configs = validity.valid_report_configs
-        warnings.append(
-            gettext_lazy('Your app contains references to reports that are '
-                         'deleted. These will be removed on save.')
-        )
+    try:
```

Comment:
I'm not sold that this is the right approach.  It seems strange that ```check_report_validity``` would fail hard when report ids belong to another domain, but not fail when the report ids don't exist at all.